paasta_tools/config_utils.py

```diff
@@ -192,13 +200,13 @@ def validate(self):
             return_code = validate_auto_config_file(filepath) and return_code
         return return_code
 
-    def commit_to_remote(self, extra_message: str = ""):
-        if not self.validate():
+    def commit_to_remote(self, extra_message: str = "", validate: bool = True):
+        if validate and not self.validate():
```

Comment:
when would we not want to validate?

Reply:
`self.validate()` will fail if we pass a file that is unexpected (not supported) by the validator. This toggle just provides an escape hatch where we don't want to provide a validator (though ideally all files will have some validation of some sort)